edn/src/edn.rustpeg

```diff
@@ -0,0 +1,106 @@
+
+use std::collections::{BTreeSet, BTreeMap, LinkedList};
+use std::iter::FromIterator;
+use types::{Pair, Value};
+use ordered_float::OrderedFloat;
+
+// Goal: Be able to parse https://github.com/edn-format/edn
+// Also extensible to help parse http://docs.datomic.com/query.html
+
+// TODO: Support tagged elements
+// TODO: Support comments
+// TODO: Support discard
+
+#[export]
+nil -> Value = "nil" {
+    Value::Nil
+}
+
+#[export]
+boolean -> Value =
+    "true" { Value::Boolean(true) } /
+    "false" { Value::Boolean(false) }
+
+digit = [0-9]
+sign = "-" / "+"
+
+#[export]
+integer -> Value = i:$( sign? digit+ ) {
+    Value::Integer(i.parse::<i32>().unwrap())
```

Comment:
Thanks @palango 
i64 fix is in https://github.com/mozilla/mentat/pull/149/commits/29bc8081a1983e9e59b1b0450bf1307a2db053b6 and BigInt addition is in https://github.com/mozilla/mentat/pull/149/commits/41f59977f67c6cfd53e88e51b1bb2975510f82be